packages/xchain-radix/src/const.ts

```diff
@@ -0,0 +1,63 @@
+import { Network, RootDerivationPaths } from '@xchainjs/xchain-client/lib'
+import { Asset } from '@xchainjs/xchain-util/lib'
+
+/**
+ * Chain identifier for Radix.
+ * This constant represents the identifier for the Radix Chain.
+ */
+export const RadixChain = 'RADIX' as const
+
+export const TransferTransactionManifest = `CALL_METHOD
+Address("account_rdx129u2xndsvv9y52y4lfw9kegnszygfus5fah9y0exawahdk83ak4xvl")
```

Comment:
I do not think that this manifest has been updated yet with the new addresses?

Also, can I suggest that we move this manifest to the `getFees` function to make it clear what its use is?

Reply:
yep, moving it makes sense. using the new address now